assets/js/components/surveys/CurrentSurvey.test.js

```diff
@@ -218,7 +230,7 @@ describe( 'CurrentSurvey', () => {
 
 		fireEvent.click( getByLabelText( 'Delighted' ) );
 
-		expect(
+		await waitFor( () =>	expect(
```

Comment:
Please, don't use tabs within the line. We use tabs only for indentation.
```suggestion
		await waitFor( () => expect(
```